python-sdk/nuscenes/scripts/export_kitti.py

```diff
@@ -81,7 +81,10 @@ def __init__(self,
             os.makedirs(self.nusc_kitti_dir)
 
         # Select subset of the data to look at.
-        self.nusc = NuScenes(version=nusc_version)
+        self.nusc = NuScenes(version=nusc_version) 
+        """ If you get error - Database not found v1.0-mini, then add datastore='path of the dataset'
+            that is, self.nusc = NuScenes(datastore='path_of_dataset', version=nusc_version)
+            Example , datastore='/home/user_name/sets/data/nuscenes' and put all the extracted folders here ) """
```

Comment:
If it's relevant to you to modify the data root, then let's make this an argument to the constructor, rather than writing a comment to modify the code.